vendor/github.com/docker/go-plugins-helpers/sdk/unix_listener_systemd.go

```diff
@@ -8,7 +8,7 @@ import (
 	"net"
 	"os"
 
-	"github.com/coreos/go-systemd/activation"
+	"github.com/coreos/go-systemd/v22/activation"
```

Comment:
This systemd dependency is going to be interesting.

Reply:
there is a tool named `mod`: https://github.com/marwan-at-work/mod which does the dirty work of updating mejor versions for you. :)